plugins/reporters/asciidoc/src/funTest/kotlin/DocBookTemplateReporterFunTest.kt

```diff
@@ -21,25 +21,25 @@ package org.ossreviewtoolkit.plugins.reporters.asciidoc
 
 import io.kotest.core.spec.style.StringSpec
 import io.kotest.engine.spec.tempdir
-import io.kotest.matchers.should
+import io.kotest.matchers.shouldBe
 
 import java.time.LocalDate
 
 import org.ossreviewtoolkit.reporter.ORT_RESULT
 import org.ossreviewtoolkit.reporter.ReporterInput
-import org.ossreviewtoolkit.utils.test.getAssetFile
-import org.ossreviewtoolkit.utils.test.matchExpectedResult
+import org.ossreviewtoolkit.utils.test.getAssetAsString
+import org.ossreviewtoolkit.utils.test.patchExpectedResult
```

Comment:
commit: Why is it simler to migrate if `patchExpectedResult()` is used?

(I wonder if keeping `matchExpectedResult` would be more consistent?)

Reply:
So, we now have the limitation of not beeing able to use `matchExpectedResult()` anymore, for projects which have been migrated. Should we rather fix that instead of migrating further things to `patchExpectedResult()` ?

Reply:
> Should we rather fix that instead of migrating further things to `patchExpectedResult()` ?

That's my plan, except for the very few places where `matchExpectedResult()` did not add value anyway, and thus were replaced with `patchExpectedResult()`.

Reply:
> except for the very few places

ok, if these are just a few. If these should turn out to be plenty, then maybe consistency can be more valuable?

Reply:
I'm not planning to do more replacements that in those 4 files of this PR, see 98c0d27b2928951ab58c662b3acd3d8c9d5457b6.